pkg/sbom/document.go

```diff
@@ -0,0 +1,117 @@
+package sbom
+
+import (
+	"context"
+	"crypto/sha1"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"chainguard.dev/apko/pkg/sbom/generator/spdx"
```

Comment:
i never quite understood why we (re)-import spdx stuff from apko side of things.
Sort of wished the two are stand-alone; as the needs to generate things in melange, are very different to the needs in apko (create "manifest" sbom + merge multiple apk sbom) versus melange's (create "apk" sbom).

Reply:
I agree in general — let's put this on the "nice to have" list for this PR's scope, and make sure we get to it soon one way or another